app/facades/foods_facade.rb

```diff
@@ -0,0 +1,17 @@
+class FoodsFacade
+  attr_reader :search_query
+
+  def initialize(food_info)
+    @search_query = food_info[:q]
+  end
+
+  def food_search
+    FoodDataService.new.food_search(@search_query)[:foods].first(10).map do |food|
+      Food.new(food)
+    end
+  end
+  
+  def total_results_from_search
+    FoodDataService.new.food_search(@search_query)[:totalHits]
```

Comment:
This is doing double the work - we're making the same APi call twice just to get the same data, but only using one piece of it here. 

Can you think of another way to get the `totalHits` and save it to a variable, maybe, in the `food_search` method instead?